include/operations/blas3_trees.hpp

```diff
@@ -169,6 +170,303 @@ inline bool do_check<false>(bool) {
   return true;
 }
 
+/*!
+ * @brief NoLocalGemmFactory is a template class whose instantiations provide
+ *        different implementations of the GEMM device function where the is no
```

Comment:
of the GEMM device function -> of the GEMM kernel

Reply:
fixed